#A representation of a multi-dof joint state

Header header

string[] joint_names
MultiDOFJointValue[] joint_values
